__tests__/app.test.js

```diff
@@ -94,4 +93,90 @@ describe("GET /api/review/:review_id", () => {
+
+describe("PATCH /api/review/:review_id", () => {
+    test("Status 200: votes are altered by the correct positive amount", () => {
+        const reviewVoteUpdate = {
+            inc_votes: 10,
+        }
+        return request(app)
+        .patch("/api/reviews/3")
+        .send(reviewVoteUpdate)
+        .expect(200)
+        .then (({body}) => {
+            expect(body.review).toEqual({
+                review_id: 3,
+                title: 'Ultimate Werewolf',
+                category: 'social deduction',
+                designer: 'Akihisa Okui',
+                owner: 'bainesface',
+                review_body: "We couldn't find the werewolf!",
+                review_img_url: 'https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png',
+                created_at: `2021-01-18T10:01:41.251Z`,
+                votes: 15,    
+            });       
+        });
+    });
+    test("Status 200: votes are altered by the correct negative amount", () => {
+        const reviewVoteUpdate = {
+            inc_votes: -10,
+        }
+        return request(app)
+        .patch("/api/reviews/3")
+        .send(reviewVoteUpdate)
+        .expect(200)
+        .then (({body}) => {
+            expect(body.review).toEqual({
+                review_id: 3,
+                title: 'Ultimate Werewolf',
+                category: 'social deduction',
+                designer: 'Akihisa Okui',
+                owner: 'bainesface',
+                review_body: "We couldn't find the werewolf!",
+                review_img_url: 'https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png',
+                created_at: `2021-01-18T10:01:41.251Z`,
+                votes: -5,    
+            });       
+        });
+    });
+    test("Status 400: something that is not a number is passed as the id in the path", () => {
+        const reviewVoteUpdate = {
+            inc_votes: -10,
+        }
+        return request(app)
+        .patch("/api/reviews/elephant")
+        .send(reviewVoteUpdate)
+        .expect(400)
+        .then(({ body }) => {
+            expect(body.msg).toBe("bad request");
+        })
+    });
+    test("Status 400: user passes something not a number in inc_votes", () => { 
+        const reviewVoteUpdate = {
+            inc_votes: 'dave',
+        }
+        return request(app)
+        .patch("/api/reviews/3")
+        .send(reviewVoteUpdate)
+        .expect(400)
+        .then(({ body }) => {
+            expect(body.msg).toBe("bad request");
+        });
+    });
+    test("Status 404: valid number in path but doesn't match a review.", () => {
+        const reviewVoteUpdate = {
+            inc_votes: 10,
+        }
+        return request(app)
+        .patch("/api/reviews/5654343")
+        .send(reviewVoteUpdate)
+        .expect(404)
+        .then(({ body }) => {
+            expect(body.msg).toBe("page not found");
+        });
+    });
+    
 });
```

Comment:
Would like to see a test for if the inc_votes key is broken/incorrect or missing entirely such as {} or {votes: 10}

Reply:
Will do now :)